integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7802,6 +7805,14 @@ msgstr "Dieser Ort wird in folgenden Kontakten verwendet."
 msgid "This location is not currently referred to in any contact."
 msgstr "Zur Zeit gibt es keine Kontakte zu diesem Ort."
 
+#: cms/templates/pois/poi_form_sidebar/related_contacts_box.html
+msgid "Create a new contact"
+msgstr "Kontakt erstellen"
+
+#: cms/templates/pois/poi_form_sidebar/related_contacts_box.html
+msgid "The new contact was successfully created."
+msgstr "Ein neuer Kontakt wurde erfolgreich erstellt."
```

Comment:
```suggestion
msgstr "Der neue Kontakt wurde erfolgreich erstellt."
```
This is very nit picky, but I think we should keep the definitive article to stay consistent to the English version, but yes, it doesn't sound as good